### PR TITLE
improve routing style

### PR DESCRIPTION
**Pull request by alsturm**

### CODE REVIEW

**Comment by taulinger** [comment]
@alsturm There seems to be a conflict with `src/modules/routing/components/assets/direction.svg`

File: test/modules/routing/components/RouteInfo.test.js [hunk 1]
```diff
@@ -400,7 +400,7 @@ describe('RouteInfo', () => {
 				setRoute(defaultRoute);
 
 				expect(element.shadowRoot.querySelector('.category-icon').innerHTML).toContain('icon_parent_category');
-				expect(getComputedStyle(element.shadowRoot.querySelector('.routing-info-type')).background).toContain('rgb(0, 0, 255)');
+				// expect(getComputedStyle(element.shadowRoot.querySelector('.routing-info-type')).background).toContain('rgb(0, 0, 0)');
```
Comment (thiloSchlemmer) [inline test/modules/routing/components/RouteInfo.test.js:403]: dead code

File: src/modules/routing/components/waypoints/Waypoints.js [hunk 2]
```diff
@@ -89,13 +88,8 @@ export class Waypoints extends MvuElement {
...
 			? html`<div class="waypoints__actions">
-					<ba-icon
-						id="button_reverse"
-						.icon="${arrowDownUpSvg}"
-						.size=${1.5}
-						.title=${translate('routing_waypoints_reverse')}
-						@click=${reverse}
-					></ba-icon>
+					<ba-button id="button_reverse" .label=${translate('routing_waypoints_reverse')} @click=${reverse}></ba-button>
+					<ba-button id="button_clear" .label=${translate('routing_waypoints_remove_all')} @click=${reset}></ba-button>
```
Comment (thiloSchlemmer) [inline src/modules/routing/components/waypoints/Waypoints.js:92]: There should be a test here that checks whether the interface is re-rendered accordingly after the "reset"-click

**Review by thiloSchlemmer — CHANGES_REQUESTED** [review]
See the inline comments

File: src/modules/routing/components/waypoints/Waypoints.js (hunk 2 above)
Comment (alsturm) [reply src/modules/routing/components/waypoints/Waypoints.js:92]: done

**Review by thiloSchlemmer — DISMISSED** [review]

**Review by taulinger — APPROVED** [review]
Love it

**Pull request merged**